relatio/narrative_models.py

```diff
@@ -683,7 +703,47 @@ def plot_clusters(
     def plot_selection_metric(self, metric: str, path=None, figsize=(14, 8)):
```

Comment:
I would add metric as optional: 

`metric: Optional[str] = None`

If the user does not specify it, we provide the most sensible default.